packages/core/lib/testing/Test.js

```diff
@@ -13,7 +13,7 @@ const SolidityTest = require("./SolidityTest");
 const RangeUtils = require("@truffle/compile-solidity/dist/compilerSupplier/rangeUtils");
 const expect = require("@truffle/expect");
 const Migrate = require("@truffle/migrate");
-const Profiler = require("@truffle/compile-solidity/dist/profiler");
+const { Profiler } = require("@truffle/compile-solidity/dist/profiler");
```

Comment:
This is an example of a change that wouldn't be necessary if you were to use the `export =` syntax in `compile-solidity/dist/profiler`

Reply:
I like named imports :)

Reply:
Yeah totally fair since this is an internal module. Just highlighted it more as an example of the difference caused by the way that TS handles exports. Apologies if this was already well understood on your end. I only recently encountered this "restriction" of the TS export syntax.